docs/recommendations.md

```diff
@@ -16,3 +16,51 @@ Always bind to localhost rather than to 0.0.0.0 or any interface, unless there i
 Binding to 0.0.0.0 or "all" interfaces can make your application respond on all current and future network interfaces.
 
 Issue: ([#19](https://github.com/open-telemetry/sig-security/issues/19#issue-1926445623))
+
+## Container images
+
+### Use the smallest image possible
+While the size of an image doesn't matter for the runtime, we are responsible for everything that is in the image. If you are delivering a statically linked binary, you can probably build a `scratch` image for production purposes. It might be a good idea to provide a second set of images for debugging purposes, such as ones including a shell and networking utilities.
+
+If you need to include files in the image, such as root certificates, use the builder pattern to obtain the files and copy only those into the final image. Example:
+
+```Dockerfile
+FROM alpine:3.20 as certificates
+RUN apk --no-cache add ca-certificates
+
+FROM scratch
+COPY --from=certificates /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY mybinary mybinary
```

Comment:
Do we have opinions regarding symbol stripping?

Reply:
I don't, if you have materials that could be relevant to us, please share and I'll take a look.